lib/utils/local-process.js

```diff
@@ -44,7 +44,7 @@ Ensure the content folder has correct permissions and try again.`));
             });
 
             // Stick the pid into the pidfile so we can stop the process later
-            fs.writeFileSync(path.join(cwd, PID_FILE), cp.pid);
+            fs.writeFileSync(path.join(cwd, PID_FILE), `${cp.pid}`);
```

Comment:
Do you know what broke here?

Reply:
Yeah so from what I was seeing Node 14 tightened up the type checking for the writeFile methods - it now errors if you pass it contents that aren't either a string or some sort of data buffer

Reply:
Aah I see. Could we do this to make it more obvious?

```suggestion
            fs.writeFileSync(path.join(cwd, PID_FILE), cp.pid.toString());
```

Reply:
Yeah the `toString` call makes sense - I'll make that change in the morning before I merge 🙂